src/components/ChatWindow/MessageList/TextMessage/TextMessage.tsx

```diff
@@ -23,6 +26,31 @@ interface TextMessageProps {
   isLocalParticipant: boolean;
 }
 
+function addLinks(text: string) {
+  const matches = linkify().match(text);
+  if (!matches) return text;
+
+  const results = [];
+  let i = 0;
+  let lastIndex = 0;
+
+  while (i < matches.length) {
```

Comment:
nit: why not a forEach loop?

Reply:
It didn't cross my mind to use a forEach loop for some reason. But a forEach loop looks cleaner here, so I added it.